west.yml

```diff
@@ -162,7 +162,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: fbbe8f22f34cadc43b69f9eb3fca10e301874e36
```

Comment:
as a good practice, I suggest moving manifest changes as a first commit or part of the first commit.

Reply:
That would mess with bissectability, changes in HAL are valid from this commit forward